filters/exclusions.txt

```diff
@@ -257,5 +257,10 @@ _campaign=$popup
 ||github.io^$third-party,xmlhttprequest,domain=~facebook.com|~gametactic.org|~github.com|~github.io|~tumblr.com|~wottactic.com
 !###########################
 !
+!### NoCoin Filter List ###
+! https://github.com/AdguardTeam/AdguardFilters/issues/84173
+/^##\.ad$/
```

Comment:
With `^` the rules like  `~example.com##.ad` will not be excluded.
Maybe better `/~.*##\.ad$/`

Reply:
I mean this rule: https://github.com/AdguardTeam/FiltersRegistry/blob/28ddd0b6c22c9d50eb7d6d1593eff80ea0461f62/filters/ThirdParty/filter_216_PolskiFiltr/filter.txt#L232

Reply:
Okay.

Reply:
I think `~` in the head should not be added.

Reply:
`/.*##\.ad$/` will exclude domain specific rules.

Reply:
> `/.*##\.ad$/` will exclude domain specific rules.

Oh, sure.